Add rcon cli (#26)
itzg/docker-bungeecord: README.md

  -e PLUGINS=https://www.example.com/plugin1.jar,https://www.example.com/plugin2.jar
  ```

* **ENABLE_RCON**

>   Enable the rcon server (that use third plugin for work [orblazer/bungee-rcon](https://github.com/orblazer/bungee-rcon), its automatically downloaded)

Reviewer: Minor wording tweak:
```suggestion
  Enable the rcon server (that uses a third-party plugin to work, [orblazer/bungee-rcon](https://github.com/orblazer/bungee-rcon), which is automatically downloaded)
```

  -p 25565:25577
  ```

## Interacting with the server

[RCON](http://wiki.vg/RCON) is enabled by default, so you can `exec` into the container to

> access the Minecraft server console:

Reviewer: copy-paste left over from minecraft server image?
```suggestion
access the Bungeecord server:
```

Author: Yes and visibly i have forgot some change
